script/deploy-canary-demo

```diff
@@ -38,101 +38,101 @@ ChimeSDKMeetings_ChimeSDKMediaPipelinesBeta)
-    npm run deploy -- -r ap-southeast-1 -b chime-sdk-meetings-$chimeSDKMediaPipelinesStackId-demo-beta-canary-ap-southeast-1$canarySuffix -s chime-sdk-meetings-demo-beta-canary-ap-southeast-1$canarySuffix -o chime-sdk-meetings-sin-demo-beta-canary$canarySuffix -p $GAMMA_CHIME_SERVICE_PRINCIPAL -c $GAMMA_CHIME_ENDPOINT -m $GAMMA_CHIME_ENDPOINT_AP_SOUTHEAST_1 -t -l --chime-sdk-media-pipelines-region ap-southeast-1 --chime-sdk-media-pipelines-endpoint GAMMA_CHIME_MEDIA_PIPELINES_ENDPOINT_AP_SOUTHEAST_1
+    npm run deploy -- -r us-east-1 -b chime-sdk-meetings-$chimeSDKMediaPipelinesStackId-demo-beta-canary-us-east-1$canarySuffix -s chime-sdk-meetings-$chimeSDKMediaPipelinesStackId-demo-beta-canary-us-east-1$canarySuffix -o chime-sdk-$chimeSDKMediaPipelinesStackId-iad-demo-beta-canary$canarySuffix -p $GAMMA_CHIME_MEDIA_PIPELINES_SERVICE_PRINCIPAL -m $GAMMA_CHIME_ENDPOINT_US_EAST_1 -t -l --chime-sdk-media-pipelines-endpoint $GAMMA_CHIME_MEDIA_PIPELINES_ENDPOINT_US_EAST_1
+    npm run deploy -- -r eu-central-1 -b chime-sdk-meetings-$chimeSDKMediaPipelinesStackId-demo-beta-canary-eu-central-1$canarySuffix -s chime-sdk-meetings-$chimeSDKMediaPipelinesStackId-demo-beta-canary-eu-central-1$canarySuffix -o chime-sdk-$chimeSDKMediaPipelinesStackId-fra-demo-beta-canary$canarySuffix -p $GAMMA_CHIME_MEDIA_PIPELINES_SERVICE_PRINCIPAL -m $GAMMA_CHIME_ENDPOINT_EU_CENTRAL_1 -t -l --chime-sdk-media-pipelines-region eu-central-1 --chime-sdk-media-pipelines-endpoint $GAMMA_CHIME_MEDIA_PIPELINES_ENDPOINT_EU_CENTRAL_1
+    npm run deploy -- -r ap-southeast-1 -b chime-sdk-meetings-$chimeSDKMediaPipelinesStackId-demo-beta-canary-ap-southeast-1$canarySuffix -s chime-sdk-meetings-$chimeSDKMediaPipelinesStackId-demo-beta-canary-ap-southeast-1$canarySuffix -o chime-sdk-$chimeSDKMediaPipelinesStackId-sin-demo-beta-canary$canarySuffix -p $GAMMA_CHIME_MEDIA_PIPELINES_SERVICE_PRINCIPAL -m $GAMMA_CHIME_ENDPOINT_AP_SOUTHEAST_1 -t -l --chime-sdk-media-pipelines-region ap-southeast-1 --chime-sdk-media-pipelines-endpoint $GAMMA_CHIME_MEDIA_PIPELINES_ENDPOINT_AP_SOUTHEAST_1
     ;;
 
 Chime_ChimeSDKMediaPipelinesBeta)
     echo "Deploying to beta stage for canary that talks to gamma Chime client for meetings and gamma ChimeSDKMediaPipelines client for media pipelines"
 
     # Uses Chime client for meetings and ChimeSDKMediaPipelines client for media pipelines
-    npm run deploy -- -r us-east-1 -b chime-sdk-$chimeSDKMediaPipelinesStackId-demo-beta-canary-us-east-1$canarySuffix -s chime-sdk-meetings-demo-beta-canary-us-east-1$canarySuffix -u false -o chime-sdk-meetings-iad-demo-beta-canary$canarySuffix -p $GAMMA_CHIME_SERVICE_PRINCIPAL -c $GAMMA_CHIME_ENDPOINT -m $GAMMA_CHIME_ENDPOINT_US_EAST_1 -t -l  --chime-sdk-media-pipelines-endpoint $GAMMA_CHIME_MEDIA_PIPELINES_ENDPOINT_US_EAST_1
```

Comment:
changes to this section include:
- adding $chimeSDKMediaPipelinesStackId- to the stack name to prevent overlap
- adding $chimeSDKMediaPipelinesStackId- to the output bucket name to prevent overlap
- specifying gamma media pipelines service principal
- removing the -c argument, as it is unused (the -m argument takes precedence in this case)